test/stage2/compare_output.zig

```diff
@@ -170,6 +170,35 @@ pub fn addCases(ctx: *TestContext) !void {
             "",
         );
     }
+
+    {
+        // TODO add a test case for comptime substraction of numbers
```

Comment:
You don't need this TODO comment here. We have a whole suite of behavioral tests (from testing stage1) and the plan is to get them all passing in stage2 as well. You can rest assured comptime subtraction of numbers is covered.